Makefile

```diff
@@ -136,7 +136,7 @@ undeploy-dev: kustomize
 	${KUSTOMIZE} build config/overlays/development | kubectl delete -f -
 
 # Generate manifests e.g. CRD, RBAC etc.
-manifests: controller-gen kustomize
+manifests: controller-gen
```

Comment:
Removing the `kustomize` dependency here seems to cause the Go build error `kustomize: not found`

```yaml
/bin/sh: 1: /home/runner/work/kserve/kserve/bin/kustomize: not found
make: *** [Makefile:169: manifests] Error 127
Error: Process completed with exit code 2.
```
https://github.com/kserve/kserve/actions/runs/6278191515/job/17051425230?pr=3141#step:6:330

Reply:
Oh, yeah. Sorry about that. It was my local change that wasn't intended to be pushed.